autoload/android.vim

```diff
@@ -243,39 +243,42 @@ function! android#setClassPath()
     return
   endif
 
-  let l:jarList = []
-  let l:srcList = []
+  if exists(":JCstart")
+    return
+  else
```

Comment:
Else after a return is not necessary and adds useless indentation and the diff larger than it needs to be. Just endif it here and keep the rest of the function without modification.